packages/block-library/src/post-time-to-read/index.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-time-to-read` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-time-to-read` block on the server.
+ *
+ * @param  array    $attributes Block attributes.
+ * @param  string   $content    Block default content.
+ * @param  WP_Block $block      Block instance.
+ * @return string Returns the rendered post author name block.
+ */
+function render_block_core_post_time_to_read( $attributes, $content, $block ) {
```

Comment:
It would be nice to write a PHPUnit test for this function.
Thanks!

Reply:
Thanks for the advice. I will give it a try 👍

Reply:
Added by cd780a2